### DISCUSSION TITLE
Is there any way to download files from the victim machine using netcat reverse_tcp?

### DISCUSSION THREAD

**Discussion by FidoHUN** [t3l3machus/Villain#137] [Q&A]
The netcat reverse_tcp payload works like a charm. Sometimes I need to tweak a little to bypass AV softwares but it's not a big deal. 

My favorite feature is the built in upload functionality.

Is there any way to download files from the victim machine when the netcat reverse shell is established?

Thanks for the help! ✌️

**Comment by t3l3machus** [comment]
Hi @FidoHUN, 
There is no built-in download functionality atm, but I intend to add it in the future. For the time being, you can use traditional ways to download files, like copy files via SMB using impacket smbserver.py, or via http, ftp, etc.

If you have other proposals for improvement or discover bugs (plenty of theme hanging around), please open an issue :)

Thanks!
